packages/base/Button/src/Button/Button.tsx

```diff
@@ -14,7 +14,10 @@ import type { Theme } from '@material-ui/core'
 import { makeStyles, ButtonBase } from '@material-ui/core'
 import { Loader } from '@toptal/picasso-loader'
 import { Container } from '@toptal/picasso-container'
-import { noop , toTitleCase } from '@toptal/picasso-utils'
+import { noop, toTitleCase } from '@toptal/picasso-utils'
+// to resolve the issue with incorrect styles between Button and Link (that uses MUI Link -> Mui Typography),
+// we need to ensure the correct order of styles import
+import '@toptal/picasso-link'
```

Comment:
```suggestion
// we need to ensure the correct order of styles import
// @TODO: to be removed when the component is migrated in FX-4614 
import '@toptal/picasso-link'
```
We might want to leave some context on when it will be removed